components/org.wso2.carbon.identity.cors.service/src/main/java/org/wso2/carbon/identity/cors/service/internal/CORSServiceHolder.java

```diff
@@ -19,17 +19,21 @@
 package org.wso2.carbon.identity.cors.service.internal;
 
 import org.wso2.carbon.identity.configuration.mgt.core.ConfigurationManager;
-import org.wso2.carbon.identity.cors.service.CORSManager;
+import org.wso2.carbon.identity.cors.mgt.core.dao.CORSConfigurationDAO;
+import org.wso2.carbon.identity.cors.mgt.core.dao.CORSOriginDAO;
+
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * Service holder class for CORS-Service.
  */
 public class CORSServiceHolder {
 
+    private List<CORSOriginDAO> corsOriginDAOS = new ArrayList<>();
+    private List<CORSConfigurationDAO> corsConfigurationDAOS = new ArrayList<>();
```

Comment:
Why we managed to add multiple configuration DAOs?

Reply:
Removed this feature.

Reply:
Removed this feature.